### ISSUE TITLE
Protect: Math Fallback is not accessible

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#6714]
When Protect's Math Fallback is displayed at the bottom of WordPress' log in form, there are no labels or indicators that you are looking at a math fallback page. This makes it difficult for folks using screen readers to actually answer the math challenge.

A potential solution is highlighted in the forum thread where this issue was reported:
https://wordpress.org/support/topic/wordpress-org-captcha-not-using-correct-structure/

**Comment by ryelle** [comment]
The input also lacks a type - it should be `type="number"`. In addition to more semantic markup and better UI on modern browser, it also allows it to be correctly styled on the chance that someone is using a plugin like Theme My Login. See this example using the dark twenty seventeen theme:

![screen shot 2017-03-29 at 12 44 09 pm](https://cloud.githubusercontent.com/assets/541093/24466053/79196402-147d-11e7-9e52-afb560757629.png)

**Comment by designsimply** [comment]
Tested and confirmed that `input` field for the math fallback in the Protect feature is not written in a `label` element.

![screen shot 2018-03-27 at tue mar 27 4 57 12 pm](https://user-images.githubusercontent.com/1119271/37999518-73c95a4e-31e0-11e8-8790-7e904b741b74.png)
<sup>Seen at http://alittletestblog.com/wp-login.php?wpe-login=true using Firefox 59.0.1 on macOS 10.13.3.</sup>

(For extra documentation) noting the main feedback here from the link @jeherve provided:

> The question to resolve should be written in a ‘label’ element and that label should be associated with the input field (for the answer).

Is this strictly required?

> The mathematical operator needs to use a plain text equivalent for people using assistive technologies because those operators are not always supported.

**Comment by KokkieH** [comment]
Related, we've had a report that typing in the box does not result in visible text.

https://twitter.com/WPThemeSpeed/status/1201695870317670400
